src/angular/src/app/spark-core-angular/components/sprk-masthead/sprk-masthead.component.ts

```diff
@@ -0,0 +1,108 @@
+import { Component, Input, HostListener } from '@angular/core';
+
+@Component({
+  selector: 'sprk-masthead',
+  template: `
+    <header [ngClass]="getClasses()" role="banner">
+      <div class="sprk-c-Masthead__content">
+        <div class="sprk-c-Masthead__top-row">
+          <div class="sprk-c-Masthead__hamburger">
+            <button class="sprk-c-Hamburger" type="button" [attr.aria-expanded]="isNarrowNavOpen ? true : false"
+            (click)="toggleNarrowNav()"
+            >
+              <span class="sprk-u-ScreenReaderText">Toggle Navigation</span>
+              <svg [ngClass]="
+                { 'sprk-c-Icon': true,
+                  'sprk-c-Icon--l': true,
+                  'sprk-c-Hamburger__icon': true,
+                  'sprk-c-Hamburger__icon--open': isNarrowNavOpen
+                }" aria-hidden="true" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 100 100">
+                <g>
+                  <path class="sprk-c-Hamburger__line sprk-c-Hamburger__line--one" d="M5 13h90v14H5z" />
+                  <path class="sprk-c-Hamburger__line sprk-c-Hamburger__line--two" d="M5 43h90v14H5z" />
+                  <path class="sprk-c-Hamburger__line sprk-c-Hamburger__line--three" d="M5 73h90v14H5z" />
+                </g>
+              </svg>
+            </button>
+          </div>
+          <div class="sprk-c-Masthead__logo">
+            <a href="#nogo">
+              <div class="drizzle-c-Logo-placeholder"></div>
+              <span class="sprk-u-ScreenReaderText">Go to the home page</span>
```

Comment:
They'll need the option to change the screenreader text "Go to the home page" - as it might not go to the home page.